modules/gappa/examineassign/meta.yml

```diff
@@ -0,0 +1,52 @@
+name: "gappa_examineassign"
+description: assigns taxonomy to query sequences in phylogenetic placement output
+keywords:
+  - sort
```

Comment:
Some appropriate keywords would be great, then it will be easier to find in https://nf-co.re/modules

Reply:
Thanks, you're absolutely right, had forgotten that!